scenes/src/app/scenes/badge/badge-scene.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {DemoMaterialModule} from '../../../../../src/assets/stack-blitz/src/app/material-module';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'app-badge-scene',
+  templateUrl: './badge-scene.html',
+  styleUrls: ['./badge-scene.scss']
+})
+export class BadgeScene {
+}
+
+@NgModule({
+  imports: [
+    DemoMaterialModule
```

Comment:
Can't we just import the badge and icon modules? the demo one has a _lot_ of stuff

Reply:
I think it auto imported the wrong one oops